scripts/auto-approve-pr.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 repo=$1
 
 get_prs() {
-    gh pr list --app "octo-sts" --label "auto-approver-bot/approve" --limit=50 --json number --jq '.[].number'
+  gh search prs --repo wolfi-dev/os --app "octo-sts" --label "auto-approver-bot/approve" --review none --state open --json number --jq '.[].number'
```

Comment:
The default limit is 30, and I think we did run into issues during busy times things were getting backed up, so we might want to bump this back to 50.